RELEASE.md

```diff
@@ -5,6 +5,11 @@ Please follow the established format:
 - Use present tense (e.g. 'Add new feature')
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
+# Upcoming Release
+
+## Major features and improvements
+
+- Kedro Viz Static Website hosting on Azure. (#1708)
```

Comment:
I would have one release note for both PRs I suppose. 

Extend support for 'Publishing and sharing Kedro-viz' to Azure and GCP.

Reply:
sure, will update this release note in the GCP PR